### ISSUE TITLE
Feature Request: Allow Spcification of Output File Name

### ISSUE THREAD

**Issue by DeveloperPaul123** [lvignoli/typst-action#2]
Currently, it's not possible to specify the output file name with the github action. It would be good to allow users to specify this or simply remove the output file name from the command so that the output `pdf` file name matches the input `typ` filename.

**Comment by lvignoli** [comment]
Yes, I almost erased the `main.pdf` in the script when fixing #1, so will fix it!

**Comment by lvignoli** [comment]
@DeveloperPaul123 continuing discussion from #3 here

In fact, if you know how you want to name the output file for every source file, it is easy enough to just rename them in another step of a workflow.

So, I would like to consider renaming the output file as a redundant feature for the action.

Do you have a use case in mind where it would be useful?

**Comment by DeveloperPaul123** [comment]
@lvignoli I think having the files be named according to the input file name should suffice for now. Just thought it would be nice since you can specify it when running `typst` directly.

**Comment by lvignoli** [comment]
Restating earlier syntax proposal of #3 here:

```text
source_file : |
   foo.typ bar.pdf
   baz.typ
   fizz.typ
   buzz.typ buzzy.pdf
```
